client/my-sites/upgrades/components/form/input.jsx

```diff
@@ -9,18 +9,17 @@ import classNames from 'classnames';
  * Internal dependencies
  */
 
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
 import FormInputValidation from 'components/forms/form-input-validation';
 import analytics from 'analytics';
-import FocusMixin from './focus-mixin';
 import scrollIntoViewport from 'lib/scroll-into-viewport';
 
 export default React.createClass( {
 	displayName: 'Input',
 
-	mixins: [ FocusMixin( 'input' ) ],
-
 	getDefaultProps() {
-		return { type: 'text', autofocus: false };
+		return { autofocus: false };
```

Comment:
Because we are not using the `type` props anymore? `FormTextInput` internally uses `type='text'`, but that's an implementation detail and we don't have to set it explicitly anymore here.